core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -150,17 +150,17 @@ public void seekTo(VideoPosition position) throws IllegalStateException {
     @Override
     public void stop() {
         reset();
+        listenersHolder.getStateChangedListeners().onVideoStopped();
```

Comment:
Sorry, this was in the reset before but actually we want the `reset()` in more places without calling the `onVideoStopped`.